docs/index.html

```diff
@@ -1859,7 +1859,7 @@ <h3 id="return-types">Return Types</h3>
 </div><div class="row">
   <div class="row-height">
     <div class="col-md-5 col-sm-12 col-height  docs-primary">
-      <p>Each of the 3 types of function should return a certain data type for use by the platform. As of core v1.0.11, there are automated checks to let you know when you&apos;re trying to pass the wrong type back. For reference, each expects:</p>
```

Comment:
I am not sure why these changes are here, I believe they happened during `husky`'s pre-commit?

Reply:
Probably because a PR was merged without running the pre-commit. But this fixes it so no worries.